pages/List/index.js

```diff
@@ -22,16 +24,19 @@ export default function List({
       fishList.map((fish) =>
         fish.id === id
           ? {
-              id,
+              id: nanoid(),
```

Comment:
please don't use nanoid inside a map: this will create a new Id for the fish that should be edited. The id is something that should stay constant over the whole lifetime of the object.